syft/frameworks/torch/nn/rnn.py

```diff
@@ -185,106 +185,77 @@ def __init__(
         # Link to issue: https://github.com/OpenMined/PySyft/issues/2500
 
         # Build RNN layers
-        self.rnn_forward = nn.ModuleList()
-        for layer in range(self.num_layers):
-            if layer == 0:
-                self.rnn_forward.append(base_cell(input_size, hidden_size, bias, nonlinearity))
-            else:
-                self.rnn_forward.append(base_cell(hidden_size, hidden_size, bias, nonlinearity))
-
+        sizes = [input_size, *[hidden_size] * (self.num_layers - 1)]
```

Comment:
I think `sizes = [input_size, *(hidden_size for _ in range(self.num_layers - 1))]` is slightly clearer but maybe it's just my opinion

Reply:
Agreed